include/dfu/dfu_target_full_modem.h

```diff
@@ -119,6 +119,15 @@ int dfu_target_full_modem_done(bool successful);
  **/
 int dfu_target_full_modem_schedule_update(int img_num);
 
+/**
+ * @brief De-initialize resources and erase the download area.
```

Comment:
```suggestion
 * @brief Deinitialize resources and erase the download area.
```
Or even uninitialize if not reset.

Reply:
I'll change to "release"